### ISSUE TITLE
Finer grain user permission

### ISSUE THREAD

**Issue by jmediamanager** [japanesemediamanager/ShokoClient#36]
**Reported by Craige1Emmott, Jan 14, 2013**

Examples
- Allow permissions to download. (Maybe include rating filter in download option).
- Update server permissions.
- Run import

The main use of this is to allow none admin users to download. I don't see why a non admin user shouldn't be allowed to download, but I do see some people not allowing users from downloading, for reasons like data usage concerns. The permissions allow for both use cases. 

Ratings would be the "hide category's" filter already in the users settings. The main step that would need to be completed before this can work is pulling the info from the torrent search results, which is in another issue ticket. Once the anime can be established from the torrent result, the categories can be pulled from anidb.net and then the hide category filter would be applied.

**Jan 20, 2013 Delete  werndlymob**

Agree with better permissions.
Although doing it via rating is impossible.
The anime data shown in the download areas, is simply a guess based on the torrent names. 

I do see where you are going with this, but I don't think the scraping would ever be reliable enough to base rules on.

**Comment by ElementalCrisis** [comment]
Will be tracked in this issue, https://github.com/japanesemediamanager/jmmserver/issues/426.
